app/src/main/java/com/example/musicapp/service/InstrumentService.kt

```diff
@@ -11,31 +11,26 @@ import android.util.Log
 
 class InstrumentService {
 
-    fun fetchInstruments() : MutableLiveData<ArrayList<Instrument>>{
+    internal fun fetchInstruments(): MutableLiveData<ArrayList<Instrument>> {
         var _instruments = MutableLiveData<ArrayList<Instrument>>()
         val service = RetrofitClientInstance.retrofitInstance?.create(MusicAppDAO::class.java)
         val call = service?.getAllInstruments()
-        call?.enqueue(object: Callback<ArrayList<Instrument>>{
+        call?.enqueue(object : Callback<ArrayList<Instrument>> {
 
             override fun onFailure(call: Call<ArrayList<Instrument>>, t: Throwable) {
                 val j = 1 + 1
                 val i = 1 + 1
-
             }
 
-
             override fun onResponse(
-
                 call: Call<ArrayList<Instrument>>,
                 response: Response<ArrayList<Instrument>>
             ) {
-            _instruments.value = response.body()
+                _instruments.value = response.body()
                 print("hi")
-                print( response.body())
+                print(response.body())
             }
-
         })
-
```

Comment:
I just removed the unnecessary blank lines here (Not one of my code review suggestions).